docs/installation/basic-install.md

```diff
@@ -1,5 +1,25 @@
 ### Basic Install
 
+After cloning this repository, you must choose the tag to run:
+```sh
+git clone git@github.com:ansible/awx-operator.git
+cd awx-operator
+git tag
+git checkout tags/<tag>
+
+# For instance:
+git checkout tags/2.7.2
+```
+
+If you work from a fork and made modifications since the tag was issued, you must provide the VERSION number to deploy. Otherwise the operator will get stuck in "ImagePullBackOff" state:
```

Comment:
It would be good to point out in a note that if they make changes to the role, they should build the image with the changes, push it to their registry, then specify the custom operator image.  or, point them to the contributing guide.

Reply:
Isn't lines 30-33 already implicitly explaining that @rooftopcellist?

> If you have a custom operator image you have built, you can specify it with:
> ```
> IMG=quay.io/$YOURNAMESPACE/awx-operator:$YOURTAG make deploy
> ````